### PR TITLE
Add rudimentary message when reevaluating in series

### PR DESCRIPTION
**Pull request by HomoCodens**
Note: Running `series` on composite models does not really seem to work yet.

### CODE REVIEW

File: R/series.R [hunk 1]
```diff
@@ -334,6 +334,12 @@ series <- function(x, series, reeval = TRUE, verbose = TRUE){
     if (reeval){
       reeval.dots <- reeval_dots(x = x$composite, series.short = series.short, verbose = FALSE)
       if (length(reeval.dots) > 0){
+        message("To speed up, extend the `seas()` call (see ?series):")
```
Comment (christophsax) [inline R/series.R:337]: It's the same code twice? Can we outfactor?

**Review by christophsax — CHANGES_REQUESTED** [review]

File: R/series.R (hunk 1 above)
Comment (HomoCodens) [reply R/series.R:337]: I thought that too but decided against it as the whole function consists of two near identical blocks at the moment. Maybe there is room for some consolidation over the whole thing?
A `dump_call_message` (or similar) helper is of course also an option.

Comment (christophsax) [reply R/series.R:337]: No need to make this bigger. I am sure the existing code has a lot of room for improvement. We can do this another day. Let's outfactor the new code for now.

**Comment by christophsax** [comment]
Muchas gracias

**Pull request merged**